apps/portals/craco.config.js

```diff
@@ -2,19 +2,26 @@ const { ProvidePlugin } = require('webpack')
 
 module.exports = {
   webpack: {
-    configure: {
-      resolve: {
-        fallback: {
-          path: require.resolve('path-browserify'),
-          timers: require.resolve('timers-browserify'),
-          fs: require.resolve('memfs'),
-          https: require.resolve('https-browserify'),
-          stream: require.resolve('stream-browserify'),
-          http: require.resolve('stream-http'),
-          events: require.resolve('events'),
-        },
-      },
-      ignoreWarnings: [/Failed to parse source map/],
+    configure: (webpackConfig) => {
+      const scopePluginIndex = webpackConfig.resolve.plugins.findIndex(
+        ({ constructor }) =>
+          constructor && constructor.name === 'ModuleScopePlugin',
+      )
+
+      webpackConfig.resolve.plugins.splice(scopePluginIndex, 1)
```

Comment:
Ran into a strange issue where a module within `@babel/runtime` was being imported from the root `node_modules` and it didn't recognize it as an external dependency, so the `ModuleScopePlugin` failed saying that the imported module wasn't in `./src`. 

It's not clear to me what change actually caused this issue to surface.

I couldn't find a real fix for this other than removing the plugin. This seemed to not cause problems for now, and I want to replace webpack with vite soon, so I think this is an acceptable stopgap.